test/e2e/serial/tests/workload_unschedulable.go

```diff
@@ -229,4 +231,140 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload unsched
 			}
 		})
 	})
+
+	// other than the other tests, here we expect all the worker nodes (including none-bm hosts) to be padded
+	Context("with zero suitable nodes", func() {
+		var numOfnrtCandidates int
+		BeforeEach(func() {
+			requiredNUMAZones := 2
+			By(fmt.Sprintf("filtering available nodes with at least %d NUMA zones", requiredNUMAZones))
+			nrtCandidates := e2enrt.FilterZoneCountEqual(nrts, requiredNUMAZones)
+
+			neededNodes := 1
+			numOfnrtCandidates = len(nrtCandidates)
+			if numOfnrtCandidates < neededNodes {
+				Skip(fmt.Sprintf("not enough nodes with 2 NUMA Zones: found %d, needed %d", len(nrtCandidates), neededNodes))
+			}
+		})
+		It("[test_id:47615][tier2][unsched] a deployment with multiple guaranteed pods resources that doesn't fit at the NUMA level", func() {
+			By("padding all the nodes")
+			requiredRes := corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("4"),
+				corev1.ResourceMemory: resource.MustParse("4G"),
```

Comment:
`G` (= 1000) or `Gi` (= 1024) ?

Reply:
I thought it affect some test results, hence I changed it to 4G, but it turns out it didn't, I'll change it back.